### ISSUE TITLE
Debug Run

### ISSUE THREAD

**Issue by rpachaly** [FLO-2DSoftware/qgis-flo-2d-plugin#1122]
In older versions there is a DEBUG Run Button, but there isn’t one in this version.  The User’s Manual shows that the button should be in the red square below, but it isn’t.

![unnamed](https://github.com/FLO-2DSoftware/qgis-flo-2d-plugin/assets/39889306/524bc8f9-d211-459e-9b0a-eb46da68a9c0)

![unnamed](https://github.com/FLO-2DSoftware/qgis-flo-2d-plugin/assets/39889306/5152c123-011b-4197-908c-d84c3de373a7)

**Comment by rpachaly** [comment]
@FLO-2DKaren 

I added the button back:
![Image](https://github.com/FLO-2DSoftware/qgis-flo-2d-plugin/assets/39889306/6b8a9487-d5c7-4f6f-ba6e-971c6f2e91e1)

I watched your video on Youtube to get acquainted with the tool, and you said that it would be better to have it docked. So, I docked the tool. You don't need to close it to use any QGIS tool:
![Image](https://github.com/FLO-2DSoftware/qgis-flo-2d-plugin/assets/39889306/67dc7307-d90d-4d2d-969d-ecbd330dff7d)

I did some small changes to the layout, nothing relevant and fixed a lot of small bugs and deprecation issues. I will create a pull request.
